Gulpfile.js

```diff
@@ -16,6 +16,15 @@ var config = {
       sprite: 'sprite.svg', //sprite name
       example: true // Build sample page
     }
+  },
+  svg: {
+    xmlDeclaration: false, // strip out the XML attribute
+    doctypeDeclaration: false, // don't include the !DOCTYPE declaration
```

Comment:
stripping these are great additions -- I'm not sure about adding root attributes below, though. This assumes that every user will want svgs to be positioned absolutely. in many cases, such as in buttons, inline-block is more ideal. I feel as though root attributes should be reserved to a per-use-case basis. 

What do you think? Is this global for all/most projects?